build_debian.sh

```diff
@@ -583,6 +584,14 @@ if [[ $CONFIGURED_ARCH == armhf || $CONFIGURED_ARCH == arm64 ]]; then
     DOCKERFS_PATH=../dockerfs/
 fi
 
+# Ensure admin gid 1000 is available
+gid_1000=$(sudo LANG=C grep -e ":1000:" $FILESYSTEM_ROOT/etc/group)
+if [ -n "${gid_1000}" ]; then
+    if [ "${gid_1000}" != "admin:x:1000:" ]; then
```

Comment:
>admin [](start = 27, length = 5)

`admin` is customizable. Use `$USERNAME` instead.
I guess you can just `id -g $USERNAME`

Reply:
Good catch! Thank you!